yaptide/application.py

```diff
@@ -16,6 +17,10 @@ def create_app():
     app = Flask(flask_name)
     logging.info("Creating Flask app %s", flask_name)
 
+    check_submodules()
```

Comment:
Why `check_submodules()` happens after creation of Flask app object ? is there any specific reason for that ?

I'd do it as the first thing in the create_app() function.

Reply:
done